Automattic/newspack-plugin
# Wizards redesign and cleanup

### All Submissions:






### Changes proposed in this Pull Request:



Part of the redesign outlined in #704. Updates:

1. Wizard navigation redesign.
2. Removal of secondary navigation – it was used only in Engagement wizard's Commenting tab, but all of it's contents can fit into one screen. A secondary navigation is an overkill.
3. Removal of `Checklist` component and PHP handlers – these were not used. 
4. Multiple refactors and smaller code & design improvements

### How to test the changes in this Pull Request:

1. Visit all the wizards
2. Observe that the functionality is the same, but layout has changed (according to the design in #704)

### Other information:
